relay/sources/rudder-pkg/makeIndex.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
```

Comment:
This should go to rudder-tools-private where there is code to manage a repository

Reply:
Yep, see here https://github.com/Normation/rudder-tools-private/pull/476